src/js/functionality.js

```diff
@@ -0,0 +1,80 @@
+import { updateLocal } from './checkbox.js';
+
+export const createTask = (taskListGet) => {
+  const elementSection = document.querySelector('.list-body');
+  elementSection.innerHTML = '';
+  const elementUl = document.createElement('ul');
+
+  taskListGet.forEach((e) => {
+    const elementLi = document.createElement('li');
+    elementLi.setAttribute('id', 'checkbox');
+    if (e.completed === true) {
+      elementLi.className = 'line';
+    } else {
+      elementLi.className = '';
+    }
+    const elementInput = document.createElement('input');
+    elementInput.id = e.id;
+    elementInput.type = 'checkbox';
+    elementInput.name = 'checkbox';
+
+    const elementSpan = document.createElement('span');
+    elementSpan.id = e.id;
+    const elementButton = document.createElement('button');
+
+    elementInput.checked = e.completed;
+    elementLi.appendChild(elementInput);
+    elementSpan.innerHTML = e.description;
+    elementLi.appendChild(elementSpan);
+    elementLi.appendChild(elementButton);
+    elementButton.id = e.id;
+    elementButton.setAttribute('onclick', 'deleteTaskFnc(this.id)');
+    elementButton.innerHTML = '<i class="fas fa-window-close"></i>';
+    elementUl.appendChild(elementLi);
+    elementSection.appendChild(elementUl);
+  });
+  updateLocal(taskListGet);
+};
+
+export const deleteAllFnc = (deletebtn, todoList) => {
+  deletebtn.addEventListener('click', () => {
+    todoList = todoList.filter((cl) => cl > 0);
+    createTask(todoList);
+    window.location.reload();
+  });
+};
+
+export function deleteTaskIm(deleteTask, todoList) {
+  const index = todoList.findIndex((prop) => prop.id === deleteTask);
+  todoList.splice(index, 1);
+  createTask(todoList);
+  window.location.reload();
+}
```

Comment:
- [ ] As it is a requirement  for this project, please add a function to clear(delete) all completed tasks, and also remember to add a button for that in the front-end. Please kindly fix that. 
- [ ] Also make sure that for each operation, the local storage is updated accordingly.